plugins/BEdita/Core/src/Model/Table/RelationsTable.php

```diff
@@ -106,15 +108,13 @@ public function validationDefault(Validator $validator)
             ->notEmptyString('name')
             ->add('name', 'unique', ['rule' => 'validateUnique', 'provider' => 'table'])
 
-            ->requirePresence('label', 'create')
-            ->notEmptyString('label')
+            ->allowEmptyString('label')
```

Comment:
maybe it's more safe to allow empty string only on create.

What happens editing a relation setting `label` or `inverse_label` as `''`?

Reply:
right, something like `->allowEmptyString('label', true, 'create)` ?

Reply:
done